### PR TITLE
add cancellationtoken info

### PR DESCRIPTION
**Pull request by MongoCaleb**
### Jira

https://jira.mongodb.org/browse/DOCSP-20211 

### Staged Changes (Requires MongoDB Corp SSO)

https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-20211/sdk/dotnet/examples/sync-changes-between-devices/#cancel-a-sync-session

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: examples/dotnet/Examples/Convert.cs [hunk 1]
```diff
@@ -25,7 +25,9 @@ public async Task TestNoSyncToSync()
             existingRealm.WriteCopy(syncConfig);
 
             // You can now delete the nonsynced realm:
-            Realm.DeleteRealm(existingConfig);
+            // :uncomment-start:
```
Comment (cbush) [inline examples/dotnet/Examples/Convert.cs:28]: Why is DeleteRealm() commented out?

Comment (MongoCaleb) [reply examples/dotnet/Examples/Convert.cs:28]: because it was borking something on the build server tests....but only intermittently.

Comment (MongoCaleb) [reply examples/dotnet/Examples/Convert.cs:28]: Doesn't change the example code.

**Comment by MongoCaleb** [comment]
+1 from Monsoor.

File: examples/dotnet/Examples/Convert.cs (hunk 1 above)
Comment (cbush) [reply examples/dotnet/Examples/Convert.cs:28]: Here's what I suggest instead: call the configuration "my.realm.nosyncToSyncExample" (or something unique to this test). Then it should not interfere with the other examples running concurrently.

The problem with uncommenting is that the commented code is not really tested and mistakes can seep in that way.

Comment (MongoCaleb) [reply examples/dotnet/Examples/Convert.cs]: fixed

**Review by cbush — APPROVED** [review]

**Comment by MongoCaleb** [comment]
Merging with the expectation that @cbush is fixing the ref issue.

**Pull request merged**